driver.py

```diff
@@ -0,0 +1,294 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import sys
+import getopt
+import os
+import time
+
+from tools import esn_prediction, optimal_values, param_string
+from optimizers import grid_optimizer
+from lorenz import generate_L96
+from sunrise import generate_elevation_series
+
+# Plot Parameters
+plt.rcParams['figure.figsize'] = (12, 9)
+plt.rcParams['figure.edgecolor'] = 'k'
+plt.rcParams['figure.facecolor'] = 'w'
+plt.rcParams['savefig.dpi'] = 400
+plt.rcParams['savefig.bbox'] = 'tight'
+plt.rcParams['text.usetex'] = True
+plt.rcParams['font.family'] = "serif"
+
+# Optimization Sets
+# radius_set = [0.5, 0.7, 0.9, 1, 1.1,1.3,1.5]
```

Comment:
why is this commented out? consider removing it.
or maybe add a comment on why that list may be preferred over the one on line 27

Reply:
I included shorter lists to execute a snakemake run in a reasonable amount of time.